Tests/test_methodbinder1.py

```diff
@@ -11,28 +11,32 @@
 from iptest import IronPythonTestCase, is_cli, is_mono, is_netcoreapp, run_test, skipUnlessIronPython
 from iptest.type_util import *
 
+from System import Int32
+
 myint1,     myint2      = myint(20),    myint(-20)
-mylong1,    mylong2     = mylong(3),    mylong(-4)
 myfloat1,   myfloat2    = myfloat(4.5), myfloat(-4.5)
 mycomplex1              = mycomplex(3)
 
+def bint(s):
```

Comment:
Maybe `def bint(*s): ...` would make a nicer call site?

Reply:
OK

Reply:
This is also a good moment to come up with a nicer name...